fix(docs) Resolve 196 mdx lint warnings for just, quickly, actually, PostgreSQL (#47358)
supabase/supabase: apps/docs/content/guides/ai/hybrid-search.mdx

Implement hybrid search in Postgres using `tsvector` (keyword search) and `pgvector` (semantic search).

> First we'll create a `documents` table to store the documents that we will search over. This is an example - adjust this to match the structure of your application.

Author: ```suggestion
First, you can create a `documents` table to store the documents that you can search over. This is an example. Adjust this to match the structure of your application.
```
